stock_picking_return_lot/wizards/stock_return_picking_line.py

```diff
@@ -13,6 +13,7 @@ class StockReturnPickingLine(models.TransientModel):
         string="Lot/Serial Number",
         domain="[('product_id', '=', product_id)]",
     )
+    origin = fields.Char(related="move_id.origin", readonly=True)
```

Comment:
Shouldn't it declared elsewhere (another module) as not really linked directly to lots ?

Reply:
In this particular case, if a picking contains more than one move for the same product and lot, it's helpful for users to see the origin of each return line in the return wizard. This helps them, for example, to identify which sale order to return.

Before this change, only a single line per product was shown in the return wizard, so the origin of the return didn't matter and wasn't necessary.